lib/full_node.js

```diff
@@ -0,0 +1,14 @@
+//      
+var bcoin = require('bcoin');
+var TrustIsRisk = require('./trust_is_risk');
```

Comment:
Popular JS libraries use - in their names. Let's use "trust-is-risk" for the npm package name? This would imply a similar name here.